initramfs/init.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/busybox sh
+#
+# SPDX-License-Identifier: AGPL-3.0+
+
+set -eu
+
+ROOT_MOUNT="/mnt/root"
+UPDATE_MOUNT="/mnt/update"
+UPDATE_SCRIPT="${UPDATE_MOUNT}/um_updater.sh"
+UPDATE_DEVICES="/dev/sd[a-z][0-9] /dev/mmcblk[0-9]p[0-9]"
+BB_BIN="/bin/busybox"
+
+init="/sbin/init"
+ro=""
+root=""
+rootflags=""
+rootfstype="auto"
+
+shutdown()
+{
+	while [ 1 ]; do
+		poweroff
+		echo "Please remove power to complete shutdown."
+		sleep 10s
+	done
+}
+
+restart()
+{
+	echo "Rebooting in 5 seconds ..."
+	sleep 5s
+	reboot
+	watchdog -T 1 -t 60 -F /dev/watchdog
+	shutdown
+}
+
+rescue_shell()
+{
+	# TODO: part 4 of EM-2396
```

Comment:
actually, don't print the splash when starting the rescue shell, but instead ...